wow/sql/address_nychastats.sql

```diff
@@ -0,0 +1,29 @@
+WITH DEVS AS (
+	SELECT 
+		DISTINCT DEVELOPMENT 
+	FROM NYCHA_BBLS_18
+	WHERE DEVELOPMENT !~ 'POLICE SERVICE AREA'
```

Comment:
A question
`!~` means `does not match regular expression, case sensitive`, right? Just wondering if the data ever would be expected to have a lowercase or mixed case version of this, in which case we might want `!~*`
https://www.postgresql.org/docs/8.1/functions-matching.html#FUNCTIONS-POSIX-TABLE

Also this is probably premature optimization but I think if the list of developments ever gets huge, using `LIKE` instead of a regex might be faster.